client/java/src/main/java/io/openlineage/client/transports/Transport.java

```diff
@@ -10,7 +10,7 @@
 import lombok.NonNull;
 
 @NoArgsConstructor
-public abstract class Transport {
+public abstract class Transport implements AutoCloseable {
```

Comment:
can we make a default method within the Transport interface instead of implementing it in files which don't need it?

Reply:
sure!